torch/_inductor/lowering.py

```diff
@@ -1064,6 +1066,96 @@ def inner_fn(idx):
     )
 
 
+@register_lowering(quantized_decomposed.quantize_per_channel, type_promotion_kind=None)
+def quantized_decomposed_quantize_per_channel(
+    input: TensorBox,
+    scales: TensorBox,
+    zero_points: TensorBox,
+    axis: int,
+    quant_min: int,
+    quant_max: int,
+    dtype: torch.dtype,
+) -> TensorBox:
+    assert len(scales.get_size()) == 1, "expect scales 1 dim"
+    assert len(zero_points.get_size()) == 1, "expect zero_points 1 dim"
+
+    if input.get_dtype() == torch.bfloat16:
```

Comment:
torch.float16 not handled on intentionally?

Reply:
Yes, since we only support `bfloat16` mixed quantization so far, `float16` will be supported in the future.